platforms/quorum/configuration/roles/create/crypto/tessera/tasks/nested_main.yaml

```diff
@@ -0,0 +1,60 @@
+# This task checks if the tm crypto material is already stored in the vault or not
```

Comment:
Dont think this file is needed now

Reply:
You're right. I will remove it and update the Readme.md as well.